src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task07.java

```diff
@@ -36,8 +36,17 @@ public static void main(String[] args) {
      * @return <code>false</code>/<code>true</code>
      */
     static boolean isLeapYear(int year) {
+        if (year % 400 == 0) {
+            return true;
+        }
+        else if (year % 100 == 0) {
+            return false;
+        } else if (year % 4 == 0) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"